modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/EditObjectField.tsx

```diff
@@ -140,21 +161,21 @@ export default function EditObjectField({
 				baseResourceURL={baseResourceURL}
 				containerWrapper={Card}
 				creationLanguageId={creationLanguageId}
+				dbObjectFieldRequired={dbObjectFieldRequired}
 				errors={errors}
 				filterOperators={filterOperators}
 				handleChange={handleChange}
-				isApproved={isApproved}
 				isDefaultStorageType={isDefaultStorageType}
 				isRootDescendantNode={isRootDescendantNode}
 				learnResources={learnResources}
-				objectDefinitionExternalReferenceCode={
-					objectDefinitionExternalReferenceCode
-				}
+				objectDefinition={objectDefinition as ObjectDefinition}
```

Comment:
Hey @igor-franca, Now you can pass the `objectDefinitionExternalReferenceCode` here as it will be used to look for the `objectDefinition` inside the `EditObjectFieldContent.tsx` component